tools/development/helpers/docs.sh

```
@@ -43,8 +47,8 @@ if [[ ! -z $1 ]] && [[ $1 == "--notebooks" ]]; then
     cd ..
 fi
 
-breathe-apidoc -o cpp_rst xml -g class
+uv run breathe-apidoc -o cpp_rst xml -g class
 
-sphinx-build -j $(nproc) -b html . _build/html
+uv run sphinx-build -j $(nproc) -b html . _build/html
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)